router/.env.example

```
@@ -12,6 +12,11 @@ CONFIG_PATH=demo.config.yaml
 #CONFIG_PATH=debug.config.yaml
 #GRAPH_CONFIG_SIGN_KEY=7kZKCz7DaLpvHKtaFEupDsBvDD9EEmUB
 
+# For local plugin development with docker-compose registry, set:
+PLUGINS_REGISTRY_URL=localhost:5050
+PLUGINS_REGISTRY_INSECURE=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PLUGINS_REGISTRY_INSECURE key should go before the PLUGINS_REGISTRY_URL key

(UnorderedKey)